src/js/global-profile.language.js

```diff
@@ -59,6 +59,11 @@
 					case 'ETH':
 						$rootScope.locale = 'am';
 						break;
+					case 'RUS':
+					case 'BEA':
+					case 'MOL':
+						$rootScope.locale = 'ru';
```

Comment:
This likely must match the .json filename or contents; 'ru-RU' or 'ru_RU', not sure which.